ui/src/app/components/rename-modal.spec.tsx

```diff
@@ -97,7 +97,7 @@ describe('should show error if new name already exist', () => {
       expect(errors.newName).toEqual(['New name already exists']);
     });
 
-    it('should return errors for duplicate R file new name case insensitive', () => {
+    it('should return errors when the new name for an R file is a case-sensitive duplicate of an existing file', () => {
```

Comment:
how about "different-case" or "alternate case" ?  I don't think case-sensitive quite captures this